ui/src/app/job-details/tabs/timing-diagram/timing-diagram.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {TaskMetadata} from "../../../shared/model/TaskMetadata";
+import { GoogleChartInterface } from 'ng2-google-charts/google-charts-interfaces';
+
+@Component({
+  selector: 'jm-timing-diagram',
+  templateUrl: './timing-diagram.component.html',
+  styleUrls: ['./timing-diagram.component.css'],
+})
+export class JobTimingDiagramComponent implements OnInit {
+  @Input() metadata: TaskMetadata[] = [];
+  timelineChart: GoogleChartInterface;
+
+  ngOnInit(): void {
+    this.buildTimelineData(this.metadata);
+  }
+
+  buildTimelineData(metadata: TaskMetadata[]): void {
+    this.timelineChart = {
+      chartType: 'Timeline',
+      dataTable: []
+    };
+    this.timelineChart.dataTable.push(['Num', 'Name', 'Start', 'End']);
+    let counter = 1;
+
+    metadata.forEach((task) => {
+      this.timelineChart.dataTable.push([counter.toString(), task.name, task.start, task.end]);
```

Comment:
Are these guaranteed to already be in a reasonable sorted order?

Reply:
I believe https://github.com/DataBiosphere/job-manager/blob/066fc41b6a4e885c058fbd5da42688d5d6662caf/servers/cromwell/jobs/controllers/jobs_controller.py#L112 sorts the tasks by start time.